src/masternode-payments.h

```diff
@@ -40,20 +40,44 @@ std::string GetRequiredPaymentsString(int nBlockHeight);
 
 class CMasternodePayee
 {
-public:
+private:
     CScript scriptPubKey;
```

Comment:
Yep, that would be better but we only have `CScript payee` in `CMasternodePaymentWinner` and switching from payee to vin would break backward compatibility. This is still quite unique though because this one corresponds to `pubKeyCollateralAddress`.